src/atoms/Icon/icons.module.scss

```diff
@@ -0,0 +1,19 @@
+@import 'src/assets/stylesheets/global';
+
+.icon-wrapper {
+  width: 100%;
```

Comment:
@drewdrewthis there's duplicate `width` and `height` in here - also this styling is making the pg icon in the navigator/ navigator 2 pages tiny